src/lit-element.ts

```diff
@@ -69,13 +69,14 @@ export class LitElement extends HTMLElement {
   }
 
   renderCallback(): TemplateResult {
+    //FBB: idea -> should we call this with `this` so you can use destructuring assignment (like preact does with state and props)
     return html``;
   }
   
   attributeChangedCallback(prop: string, _oldValue: string, newValue: string) {
-    const { type: typeFn } = this.constructor.properties[prop];
-
+    const { type: typeFn } = (this.constructor as any).properties[prop];
     if (typeFn.name === 'Boolean') {
+      //FBB: I believe there is a bug here! 
```

Comment:
below or above? :-)

Reply:
below... if you remove the attribute the property value is supposed go to false right?

Reply:
You are right... should be === prop or === "". When removing it becomes null